docs/markdown/snippets/introspect_buildoptions_no_bd.md

```diff
@@ -0,0 +1,7 @@
+## `introspect --buildoptions` can now be used without configured build directory
+
+It is now possible to run `meson introspect --buildoptions /path/to/meson.build`
+without a configured build directory.
+
+Running `--buildoptions` without a build directory produces the same output as running
+it with a freshly configured build directory.
```

Comment:
I think we need to update the snippet with an caveat about this outputting the options of all subprojects even when they will not be active after configuration. (And that IDEs should be careful about displaying subproject options without user request)